src/peerconnection.cc

```diff
@@ -248,51 +248,51 @@ void PeerConnection::OnRenegotiationNeeded() {
 
 NAN_METHOD(PeerConnection::New) {
   TRACE_CALL;
-  NanScope();
+  Nan::HandleScope scope;
 
-  if(!args.IsConstructCall()) {
-    return NanThrowTypeError("Use the new operator to construct the PeerConnection.");
+  if(!info.IsConstructCall()) {
+    return Nan::ThrowTypeError("Use the new operator to construct the PeerConnection.");
   }
 
   PeerConnection* obj = new PeerConnection();
-  obj->Wrap(args.This());
+  obj->Wrap(info.This());
 
   TRACE_END;
-  NanReturnValue(args.This());
+  info.GetReturnValue().Set(info.This());
 }
 
 NAN_METHOD(PeerConnection::CreateOffer) {
   TRACE_CALL;
-  NanScope();
+  Nan::HandleScope scope;
 
-  PeerConnection* self = ObjectWrap::Unwrap<PeerConnection>( args.This() );
+  PeerConnection* self = Nan::ObjectWrap::Unwrap<PeerConnection>( info.This() );
 
   self->_jinglePeerConnection->CreateOffer(self->_createOfferObserver, NULL);
 
   TRACE_END;
-  NanReturnValue(NanUndefined());
+  info.GetReturnValue().Set(Nan::Undefined());
 }
 
 NAN_METHOD(PeerConnection::CreateAnswer) {
   TRACE_CALL;
-  NanScope();
+  Nan::HandleScope scope;
 
-  PeerConnection* self = ObjectWrap::Unwrap<PeerConnection>( args.This() );
+  PeerConnection* self = Nan::ObjectWrap::Unwrap<PeerConnection>( info.This() );
 
   self->_jinglePeerConnection->CreateAnswer(self->_createAnswerObserver, NULL);
 
   TRACE_END;
-  NanReturnValue(NanUndefined());
+  info.GetReturnValue().Set(Nan::Undefined());
 }
 
 NAN_METHOD(PeerConnection::SetLocalDescription) {
   TRACE_CALL;
-  NanScope();
+  Nan::HandleScope scope;
 
-  PeerConnection* self = ObjectWrap::Unwrap<PeerConnection>( args.This() );
-  v8::Local<v8::Object> desc = v8::Local<v8::Object>::Cast(args[0]);
-  v8::String::Utf8Value _type(desc->Get(NanNew("type"))->ToString());
-  v8::String::Utf8Value _sdp(desc->Get(NanNew("sdp"))->ToString());
+  PeerConnection* self = Nan::ObjectWrap::Unwrap<PeerConnection>( info.This() );
+  v8::Local<v8::Object> desc = v8::Local<v8::Object>::Cast(info[0]);
+  v8::String::Utf8Value _type(desc->Get(Nan::New("type").ToLocalChecked())->ToString());
+  v8::String::Utf8Value _sdp(desc->Get(Nan::New("sdp").ToLocalChecked())->ToString());
 
   std::string type = *_type;
   std::string sdp = *_sdp;
```

Comment:
Following the mysterious `true` deeper, it seems the description `peer.createOffer` creates looks like `RTCSessionDescription { type: 'offer', sdp: true }`.